src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Performance/UseSpanClearInsteadOfFill.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using Analyzer.Utilities;
+using Analyzer.Utilities.Extensions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Microsoft.NetCore.Analyzers.Performance
+{
+    using static MicrosoftNetCoreAnalyzersResources;
+
+    /// <summary>
+    /// CA1851: Use Span.Clear instead of Span.Fill(default)
+    /// </summary>
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
```

Comment:
Wasn't it decided to drop VB support?

Reply:
It was not clear what "support" means to me. There should never be diagnostics reported for VB, but I want to be consistent with other analyzers too.

Reply:
> There should never be diagnostics reported for VB

In that case, there shouldn't be `LanguageNames.VisualBasic`.

@mavasani Is adding a suppression like the following good enough?

https://github.com/dotnet/roslyn-analyzers/blob/793113a41d7f21b03470521bf48438f2abd9b12f/src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/QualityGuidelines/DoNotDeclareEventFieldsAsVirtual.cs#L13-L15

Or is it better to have an abstract analyzer in `Core` and a concrete one in `CSharp`?

Reply:
@Youssef1313 Either way sounds good to me. Generally we follow the latter approach.